### DISCUSSION TITLE
Specify specific display for screenEdgeGapBottom setting

### DISCUSSION THREAD

**Discussion by Jon-edge** [rxhanson/Rectangle#747] [Ideas]
Currently, the defaults.screenEdgeGapBottom setting applies to all monitors. This does not work well with taskbars/docks when using two monitors with different resolutions (in my case, laptop and large secondary displays).

Implement a way to specify screen gaps by monitor.

**Comment by rxhanson** [comment]
I probably won't add this in myself, but have no reason to turn down a good PR for it.
